workspace/data-proxy-sdk/src/config.ts

```diff
@@ -22,17 +25,23 @@ export const defaultConfig: Record<Environment, DataProxyOptions> = {
 		rpcUrl: "https://rpc.seda.xyz",
 		explorerUrl: "https://explorer.seda.xyz",
 		privateKey: Buffer.from([]),
+		fastMaxProofAgeMs: 1000 * 60 * 5, // 5 minutes
+		fastAllowedClients: ["invalid-client"],
```

Comment:
Sorry to be annoying but with the latest change it's probably better to leave this empty by default?